python/pyspark/mllib/util.py

```diff
@@ -169,6 +175,28 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to
+        the end of the input vector.
+        """
+        vec = _convert_to_vector(data)
+        if isinstance(vec, SparseVector):
+            l = scipy.sparse.csc_matrix(np.append(vec.toArray(), 1.0))
```

Comment:
You won't be able to use scipy here since the user might not have it available (but I also don't think there's a need to).  It should be straightforward to get the size, indices, and values, modify them to append a 1.0, and create a new SparseVector.